### PR TITLE
Bottling Machine Conveyor Sync

### LINKED ISSUE #4313: [MC1.15 | IE119][minor] Bottling Machine render inconsistency
The bottling machine's lifting fork is set below the conveyor level when not in use (see picture).
However, at the start and finish of the lifting animation, the fork jumps from being set below to conveyorlevel and vice versa, instead of lifting from where it is resting. Don't know if the resting position is  wrong or the animation's border-position, but there you have it.
grafik

### PR DESCRIPTION
**Pull request by MalkContent**
syncs movement of items on the Bottling Machine's conveyor belt with the belt's animation
doubles the machine's speed in the process

some code cleanup/restructuring

based on: https://github.com/BluSunrize/ImmersiveEngineering/pull/4322
because of that, also fixes: https://github.com/BluSunrize/ImmersiveEngineering/issues/4313

### CODE REVIEW

**Comment by BluSunrize** [comment]
I still need to check this one and test it, then I'll do a review.

**Review by BluSunrize — APPROVED** [review]
LGTM

File: src/main/java/blusunrize/immersiveengineering/common/blocks/metal/BottlingMachineTileEntity.java [hunk 1]
```diff
@@ -290,7 +289,8 @@ public int getProcessQueueMaxLength()
 	@Override
 	public float getMinProcessDistance(MultiblockProcess<MultiblockRecipe> process)
 	{
-		return .5f;
+		float maxTicks = BottlingProcess.getMaxProcessTick();
+		return 1f-(BottlingMachineRenderer.getTransportTime(maxTicks)+BottlingMachineRenderer.getLiftTime(maxTicks))/maxTicks;
```
Comment (malte0811) [inline src/main/java/blusunrize/immersiveengineering/common/blocks/metal/BottlingMachineTileEntity.java]: I would move these functions to the TE. I suspect that the current implementation crashes on dedicated servers, and even if it doesn't I would avoid referencing the tile render class in server-side code.

File: src/main/java/blusunrize/immersiveengineering/common/blocks/metal/BottlingMachineTileEntity.java [hunk 2]
```diff
@@ -423,7 +423,9 @@ public boolean processStep(BottlingMachineTileEntity tile)
 			if(tile.energyStorage.extractEnergy(energyExtracted, true) >= energyExtracted)
 			{
 				tile.energyStorage.extractEnergy(energyExtracted, false);
-				if(++processTick==(int)(maxProcessTick*.4375))
+				processTick++;
+				float transformationPoint = BottlingMachineRenderer.getTransportTime(maxProcessTick)+BottlingMachineRenderer.getLiftTime(maxProcessTick);
```
Comment (malte0811) [inline src/main/java/blusunrize/immersiveengineering/common/blocks/metal/BottlingMachineTileEntity.java]: Same here

**Review by malte0811 — CHANGES_REQUESTED** [review]

File: src/main/java/blusunrize/immersiveengineering/common/blocks/metal/BottlingMachineTileEntity.java (hunk 1 above)
Comment (MalkContent) [reply src/main/java/blusunrize/immersiveengineering/common/blocks/metal/BottlingMachineTileEntity.java]: good point, done

**Pull request merged**